.github/workflows/sidecar-check.yml

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2020-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Sidecar Digest validation
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *"
```

Comment:
is every day at midnight a good time for this? Just wondering what other processes might be running in parallel ... prolly ok, depending on which TZ this is.